src/services/TaskService.ts

```
@@ -515,6 +515,75 @@ export const TaskService = {
+                      )
+                    : null;
+                const tags = dbTask.task_tags?.map((tt: any) => tt.tags as Tag).filter(Boolean) || [];
+                const people = dbTask.task_people?.map((tp: any) => tp.people as Person).filter(Boolean) || [];
+                return mapDbTaskToTask(dbTask as DbTask, recurrenceRule, tags, people);
+            });
+
+        } catch (error) {
+            console.error('[TaskService.getTasksContributingToEffortOnDate] Unexpected error:', error);
+            throw new Error('Failed to fetch tasks contributing to effort on date.');
+        }
+    },
```

Biome (1.9.4)

[error] This method named getTasksContributingToEffortOnDate is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this method named getTasksContributingToEffortOnDate


(lint/suspicious/noDuplicateObjectKeys)